src/Components/Navbar/AdminNavbar.js

```diff
@@ -33,7 +33,7 @@ export default function AdminNavbar(props) {
         <NavbarBrand href='/'>Admin Dashboard
         </NavbarBrand>
         <div id='admin-toggler'>
-          <DarkMode />
+          {/* <DarkMode /> */}
```

Comment:
why is this commented out?

Reply:
there's no dark mode on the new design :/